pyproject.toml

```
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel", "numpy", "Cython"]
```

Add newline at the end of file
```suggestion
requires = ["setuptools", "wheel", "numpy", "Cython"]

```